app/controllers/idv/phone_question_controller.rb

```diff
@@ -16,20 +16,16 @@ def show
     end
 
     def phone_with_camera
-      analytics.idv_doc_auth_phone_question_submitted(
-        **analytics_arguments.
-                merge(phone_with_camera: true),
-      )
+      idv_session.phone_with_camera = true
```

Comment:
@amirbey , so basically user is presented(based on bucket) this question, and answered with true or false. If user is not presented with question, this flag will be nil.

Reply:
correct 👍🏿